docs/reference.rst

```diff
@@ -0,0 +1,39 @@
+Reference
+=============
+
+The Null data type
+---------------------
+
+.. automodule:: spyql.nulltype
+   :members:
+
+
+General SQL functions
+--------------------------------
+
+.. automodule:: spyql.sqlfuncs
+   :members:
+
+
+Aggregate functions
+--------------------------------
+
+.. automodule:: spyql.agg
+   :members:
+
+
+Dictionaries
+--------------------------------
+
+.. autoclass:: spyql.qdict.qdict
+   :members:
+
+
```

Comment:
I don't know if maybe these weren't included yet but in https://spyql.readthedocs.io/en/docs/reference.html they aren't being rendered.

Reply:
fixed!